### PR TITLE
KHADRI-3-Simple-Save-Operations

### PR DESCRIPTION
**Pull request by Babjanshake**
Please Verify And Approve Sir.

### CODE REVIEW

File: src/com/khadri/hibernate/college/client/CollegeMain.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package com.khadri.hibernate.college.client;
```
Comment (khadri2021) [inline src/com/khadri/hibernate/college/client/CollegeMain.java]: change package from com.khadri.hibernate.college.client to com.khadri.hibernate.client

File: src/com/khadri/hibernate/college/client/CollegeMain.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import java.util.stream.Collectors;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.cfg.Configuration;
+
+import com.khadri.hibernate.college.entity.College;
+
+public class CollegeMain {
```
Comment (khadri2021) [inline src/com/khadri/hibernate/college/client/CollegeMain.java]: Client2

File: src/com/khadri/hibernate/college/entity/College.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package com.khadri.hibernate.college.entity;
```
Comment (khadri2021) [inline src/com/khadri/hibernate/college/entity/College.java]: change package from com.khadri.hibernate.college.entity to com.khadri.hibernate.entity

File: src/com/khadri/hibernate/college/entity/College.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+public class College {
+	private Integer collegeId;
```
Comment (khadri2021) [inline src/com/khadri/hibernate/college/entity/College.java]: change collegeId to clgId

File: src/com/khadri/hibernate/college/entity/College.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+	private String collegeName;
```
Comment (khadri2021) [inline src/com/khadri/hibernate/college/entity/College.java]: change collegeName to clgName

File: src/map.xml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<hibernate-mapping>
```
Comment (khadri2021) [inline src/map.xml]: change file name from map.xml to college-mapping.xml

File: src/com/khadri/hibernate/college/client/CollegeMain.java (hunk 1 above)
Comment (Babjanshake) [reply src/com/khadri/hibernate/college/client/CollegeMain.java]: Changed Successfully.

File: src/com/khadri/hibernate/college/client/CollegeMain.java (hunk 2 above)
Comment (Babjanshake) [reply src/com/khadri/hibernate/college/client/CollegeMain.java]: Changed Successfully.

File: src/com/khadri/hibernate/college/entity/College.java (hunk 3 above)
Comment (Babjanshake) [reply src/com/khadri/hibernate/college/entity/College.java]: Changed Successfully.

File: src/com/khadri/hibernate/college/entity/College.java (hunk 4 above)
Comment (Babjanshake) [reply src/com/khadri/hibernate/college/entity/College.java]: Changed Successfully.

File: src/com/khadri/hibernate/college/entity/College.java (hunk 5 above)
Comment (Babjanshake) [reply src/com/khadri/hibernate/college/entity/College.java]: Changed Successfully Sir.

File: src/map.xml (hunk 6 above)
Comment (Babjanshake) [reply src/map.xml]: Changed Successfully.

**Comment by khadri2021** [comment]
review don

**Review by khadri2021 — APPROVED** [review]
Approved.

**Pull request merged**